fdbcli/ConfigureCommand.actor.cpp

```diff
@@ -276,6 +276,10 @@ ACTOR Future<bool> configureCommandActor(Reference<IDatabase> db,
 		fprintf(stderr, "ERROR: A cluster cannot change its tenant mode while part of a metacluster.\n");
 		ret = false;
 		break;
+	case ConfigurationResult::ENCRYPTION_DATA_AT_REST_MODE_ALREADY_SET:
```

Comment:
Is this generated anywhere?

Reply:
@ajbeamon seems like a fallout from my PR, I intent was to update `changeConfig` routine to check if `oldValue` != `newValue` and disallow the configuration change.